### ISSUE TITLE
`netFrameworkVersion` of Function App created by CLI

### ISSUE THREAD

**Issue by t3kot3ko** [Azure/azure-cli#25087]
## Describe the bug

**Command Name**
`az functionapp create`

`netFrameworkVersion` of Function App created by using Azure CLI is different from that created by using Azure Portal.
When creating Function App by using Azure Portal with following parameters, `netFrameworkVersion` is set to "v6.0"
![image](https://user-images.githubusercontent.com/490274/211511573-b2b47d06-5a98-46ed-8949-5650c6a9b028.png)

However, it's set to "v4.0" when creating Function App by using Azure CLI like follows, even though I set `--runtime-version` option
```
az functionapp create --name {} --os-type Windows --resource-group {} --runtime dotnet --runtime-version 6.0 --storage-account {} --functions-version {} --plan {} --subscription {} --vnet {} --subnet {} --debug --verbose
```
I confirmed `netFrameworkVersion` with following command:
```
az functionapp config show --name {} --resource-group {} --query "netFrameworkVersion"
```

Moreover, I confirmed `netFrameworkVersion` was updated after performing the following command:
```
az functionapp config set --net-framework-version v6.0 --name {} --resource-group {} 
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- Create Resource Group, Storage Account, App Service Plan and VNet/Subnet in advance.
- `az functionapp create --name {} --os-type Windows --resource-group {} --runtime dotnet --runtime-version 6.0 --storage-account {} --functions-version {} --plan {} --subscription {} --vnet {} --subnet {} --debug --verbose`

## Expected Behavior
`netFrameworkVersion` is updated according to --runtime-version option when creating Function App by Azure CLI (i.e. `netFrameworkVersion` is set to "v6.0" when creating Function App with `az functionapp create ... --runtime dotnet --runtime-version 6.0`).

## Environment Summary
```
Linux-5.15.79.1-microsoft-standard-WSL2-x86_64-with-glibc2.35, Ubuntu 22.04.1 LTS
Python 3.10.8
Installer: DEB

azure-cli 2.43.0 *

Extensions:
containerapp 0.3.13
log-analytics 0.2.2
ssh 1.1.3

Dependencies:
msal 1.20.0
azure-mgmt-resource 21.1.0b1
```

**Comment by yonzhan** [comment]
route to CXP team

**Comment by SaurabhSharma-MSFT** [comment]
@t3kot3ko I am able to reproduce this issue and checking internal on the same.

**Comment by kamperiadis** [comment]
Hello @t3kot3ko Thank you for reporting this. I was able to reproduce the issue as well. This is definitely Azure CLI specific. We have added it to our backlog and will work on fixing this.

**Comment by motoko89** [comment]
Any update on this? It's been 3 months

**Comment by kamperiadis** [comment]
This will be fixed with #28825

**Comment by kamperiadis** [comment]
Marking as resolved since the fixes have already been released.
